mayastor/src/lvs/lvol.rs

```diff
@@ -238,7 +238,7 @@ impl Lvol {
                         name: self.name(),
                     }
                 })?;
-            let buf = hdl.dma_malloc(1 << 12).map_err(|e| {
+            let buf = hdl.dma_malloc(1 << 16).map_err(|e| {
```

Comment:
question: Even if this block size is changed, Will the actual IO to the disk be still done in 4k blocks?

Reply:
It's not the block size that is being changed, rather the size of the buffer that is being passed to SPDK. SPDK can handle buffers up to a size of `SPDK_BDEV_LARGE_BUF_MAX_SIZE` which is 64KB (https://github.com/spdk/spdk/blob/49911c47fb9bdae96cecd6a3f299cf87c4fba200/include/spdk/bdev.h#L56).

How the I/Os get submitted to the underlying disk is up to SPDK, but by increasing the buffer size we are able to make fewer calls into SPDK and let them perform work against this larger buffer.

Note: Rebuild also uses this buffer size to improve performance.

Reply:
Should we actually use `SPDK_BDEV_LARGE_BUF_MAX_SIZE` ?

Reply:
Good idea - done.